src/libelektra/mount.c

```diff
@@ -444,6 +444,8 @@ Backend* elektraMountGetBackend(KDB *handle, const Key *key)
 {
 	if (!key || !strcmp(keyName(key), "")) return handle->defaultBackend;
 
+	if (!handle) return NULL; // TODO: what is the best way to handle this
```

Comment:
interesting hint: did libgetenv work on Mac OS X? Do you have any idea how it can be reproduced (that handle==NULL)?